charts/posthog/templates/plugins-ingestion-deployment.yaml

```diff
@@ -1,3 +1,18 @@
 {{ if .Values.pluginsIngestion.enabled }}
+{{/*
+    The ingestion deployments handle processing of analytic events e.g. captured
+    via the `/e/` and similar endpoints.
+
+    To allow for near real time processing of most of the events, we will divert
+    some events that break certain criteria e.g. sending 1000s of events per
+    second with the same distinct id to a separate consumer to be processed.
+    This is consumer is run by the plugins-ingestion-overflow deployment and can
+    be scaled independently.
+
+    NOTE: before the PostHog app supports the `ingestion-overflow` mode, we'll
```

Comment:
Ooops this doesn't work 🤔  the plugin-server will not know what to do for ingestion-overflow

Reply:
If we want to generalise this with async consumer then we can have two params:
1. topic to read
2. pipeline to use

then for 
1. async consumer: it's async topic and async pipeline
2. ingestion: it's ingestion topic and ingestion pipeline (with overflow)
3. ingestion_overflow topic and ingestion pipeline (after overflow step)